src/libtexture/imagecache.cpp

```diff
@@ -467,6 +470,7 @@ ImageCacheFile::open (ImageCachePerThreadInfo *thread_info)
     // Since each subimage can potentially have its own mipmap levels,
     // keep track of the highest level discovered
```

Comment:
Yeah, and in particular, the case where the file itself has changed! (Like, between interactive re-renders, they update the texture.)